src/screens/CreateCommunity/index.js

```diff
@@ -22,7 +22,8 @@ import {ProgressBar} from '../../components/ProgressBar';
 import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {setCapitalFirstLetter} from '../../utils/Utils';
 import {NavigationConstants} from '../../utils/constants';
-import {verifyCommunityName} from '../../service/topics';
+import {submitCommunityName, verifyCommunityName} from '../../service/topics';
+import DiscoveryRepo from '../../service/discovery';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `submitCommunityName` is a new addition and seems to be used correctly in the code. However, it's important to ensure that this function is properly tested and handles all possible edge cases since it interacts with external services.
```diff
+ import {submitCommunityName, verifyCommunityName} from '../../service/topics';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->